Core/AppRuntime/Include/Babylon/AppRuntime.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <Babylon/JsRuntime.h>
+
+#include <memory>
+#include <string>
+
+namespace Babylon
+{
+    class TaskChain;
+
+    class AppRuntime final : public JsRuntime
+    {
+    public:
+        AppRuntime(const char* rootUrl = nullptr);
+        ~AppRuntime();
+
+        const std::string& GetRootUrl() const;
```

Comment:
We should talk about the convention for getters in our code.

```suggestion
        const std::string& RootUrl() const;
```

Reply:
Since it's `const` anyway, maybe we should just make the member public.